media/formats/BUILD.gn

```diff
@@ -4,6 +4,10 @@
 
 import("//media/media_options.gni")
 
+if (is_cobalt) {
```

Comment:
Shouldn't this be `use_starboard_media` ?

Reply:
No, this is correct—that said, I don't understand where it is used in this file

Reply:
Removed.